From 5d2955e7031825f7a78f1859ee9eae3dd844a83a Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Tue, 3 Jul 2018 16:59:55 +0800
Subject: [PATCH 4235/5242] MLK-18724-1 clk: imx8mm: remove
 IMX8MM_CLK_NAND_USDHC_BUS_CG out from
 clks_init_on[]

commit  42790de7ae6b53635edc595dc2bb0b9673811e90 from
https://source.codeaurora.org/external/imx/linux-imx.git

No need to enable IMX8MM_CLK_NAND_USDHC_BUS_CG during the imx8mm clock driver
init, so remove it from the clks_init_on[].

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
(cherry picked from commit 7a8f9c1917dec30fc37b6b8ea74461e80ecdbc30)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/imx/clk-imx8mm.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/clk/imx/clk-imx8mm.c b/drivers/clk/imx/clk-imx8mm.c
index 5aaca6f..d05e14e 100644
--- a/drivers/clk/imx/clk-imx8mm.c
+++ b/drivers/clk/imx/clk-imx8mm.c
@@ -382,7 +382,7 @@ enum {
 static int const clks_init_on[] __initconst = {
 	IMX8MM_CLK_AHB_CG, IMX8MM_CLK_DRAM_CORE,
 	IMX8MM_CLK_NOC_CG, IMX8MM_CLK_NOC_APB_CG,
-	IMX8MM_CLK_USB_BUS_CG, IMX8MM_CLK_NAND_USDHC_BUS_CG,
+	IMX8MM_CLK_USB_BUS_CG,
 	IMX8MM_CLK_MAIN_AXI_CG, IMX8MM_CLK_AUDIO_AHB_CG,
 	IMX8MM_CLK_DRAM_APB_DIV, IMX8MM_CLK_A53_DIV,
 	IMX8MM_ARM_PLL_OUT, IMX8MM_CLK_DISP_AXI_CG,
-- 
1.7.9.5

